<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,1,0,0,0">
    <item row="4" column="7">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Jump to Solution</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="nextStepButton">
      <property name="text">
       <string>Next step</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="4" column="6">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="4" column="5">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="4" column="8">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_9"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="queensLineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="boardSizeSpinBox">
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Queen positions</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Board size</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Algorithm</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QRadioButton" name="radioButton_BT">
         <property name="text">
          <string>BT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_TS">
         <property name="text">
          <string>TS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_SA">
         <property name="text">
          <string>SA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_GA">
         <property name="text">
          <string>GA</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
